pom.xml

```diff
@@ -73,7 +73,7 @@
     <bigquery.connector.hadoop2.version>hadoop2-1.0.0</bigquery.connector.hadoop2.version>
     <commons.codec.version>1.4</commons.codec.version>
     <cdap.version>6.2.0</cdap.version>
-    <cdap.plugin.version>2.4.2</cdap.plugin.version>
+    <cdap.plugin.version>2.4.4-SNAPSHOT</cdap.plugin.version>
```

Comment:
why is this change needed? usually we do not upgrade it unless there is an api change

Reply:
As mentioned in the other PR, some of the changes in my other PR for hydrator plugins are needed here.

The snapshot dependency also signals the need to release the core plugins before this dependency can be tagged with a new patch release.